src/index.ts

```diff
@@ -1,61 +1,87 @@
-import * as dotEnv from 'dotenv'
-import { expand } from 'dotenv-expand'
+import * as dotEnv from "dotenv";
+import { expand } from "dotenv-expand";
+import yargs from "yargs";
+import { Client, fetchExchange } from "@urql/core";
 
-import { logger } from './logger'
-import { buildConfig } from './config'
-
-import yargs from 'yargs'
+import { logger } from "./logger";
+import { buildConfig } from "./config";
+import { MercuryClient } from "./service/mercury";
+import { initApiServer } from "./route";
 
 interface CliArgs {
-  env: string
-  port: number
+  env: string;
+  port: number;
 }
 
 async function main() {
-  const _config = dotEnv.config({ path: '.env' })
-  expand(_config)
+  const _config = dotEnv.config({ path: ".env" });
+  expand(_config);
 
-  const config = _config.parsed || {}
-  // @ts-ignore
-  const conf = buildConfig(config)
+  const config = _config.parsed || {};
+  const conf = buildConfig(config);
 
   const argv = yargs(process.argv).options({
     env: {
-      alias: 'e',
-      type: 'string',
-      description: 'env - production or development'
+      alias: "e",
+      type: "string",
+      description: "env - production or development",
     },
     port: {
-      alias: 'p',
-      type: 'number',
-      description: 'port for server'
+      alias: "p",
+      type: "number",
+      description: "port for server",
+    },
+  }).argv as CliArgs;
+
+  const env = argv.env || "development";
+  const port = argv.port || 3002;
+
+  const client = new Client({
+    url: `${conf.mercuryUrl}:5000/graphql`,
```

Comment:
similar question as above - should we make the port configurable or are we confident that this won't change?

Reply:
These ports have been standard so far but I can pull them out.

Reply:
Yeah, graphQL ports won't change.